docs/source/reference/api-server/examples/api-server-gpu-metrics-setup.rst

```diff
@@ -23,6 +23,49 @@ requirements:
 * **DCGM-Exporter** is running on the cluster and exposes metrics on
   port ``9400``.  Most GPU Operator installations already deploy DCGM-Exporter for you.
 
+.. dropdown:: Service for the DCGM-Exporter Pods
+
+    Ensure that a Kubernetes Service for DCGM-Exporter exists and includes the following annotations so Prometheus can scrape its metrics:
```

Comment:
We should add a script to check the existence, and say if it does not, what to do.

Reply:
Revised the doc, PTAL again.